src/actions/alerts.js

```diff
@@ -0,0 +1,17 @@
+import C from '../reducers/constants';
+
+export function addMessage({ message, duration, severity }) {
```

Comment:
To me, `addAlert` (and `deleteAlert`) would make more sense than `addMessage`, but it's not that important.